### DISCUSSION TITLE
Problem transferring files between IOS / Android / Android TV

### DISCUSSION THREAD

**Discussion by kisuro** [localsend/localsend#1151] [Discussion (English, International)]
Hi. 
Can you tell me what could be the problem? 
I successfully see and send files between Android-Android, IOS-IOS , BUT when trying to send between different OS devices are not visible. For example in the IOS app I can't see the Android app running.
All devices are on the same network. 

Thanks

**Comment by Tienisto** [comment]
You can try to tap on "Sync" on both devices

**Comment by Otus159** [comment]
I think there should be be an app for. Android tv also
